control/modelsimp.py

```diff
@@ -50,7 +50,7 @@
 from .statesp import StateSpace
 from .statefbk import gram
 
-__all__ = ['hsvd', 'balred', 'modred', 'era', 'markov', 'minreal']
+__all__ = ['hsvd', 'balred', 'modred', 'era', 'markov', 'minreal',]
```

Comment:
Can you delete it, then? I am writing additional comments now, so you might want to wait before rebasing until I send them all.